app/models/manageiq/providers/openshift/container_manager/container_image.rb

```diff
@@ -1,10 +1,48 @@
 class ManageIQ::Providers::Openshift::ContainerManager::ContainerImage < ContainerImage
-  def annotate_deny_execution(causing_policy)
+  def annotate_image(annotations)
     ext_management_system.annotate(
       "image",
       digest,
+      annotations
+    )
+  end
+
+  def openscap_summary
+    failed_rules = openscap_rule_results.where(:result => "fail").group(:severity).count
+    [[['High'], 'Critical', 3],
+     [['Medium'], 'Important', 2],
+     [['Low'], 'Medium', 1],
+     [['Info', 'Unknown'], 'Low', 0]].collect do |severities, label, index|
+      {
+        :label         => label,
+        :severityIndex => index,
+        :data          => failed_rules.select { |sev| severities.include?(sev) }.values.sum
+      }
+    end
+  end
+
+  def security_quality_annotation(compliant)
+    {"quality.images.openshift.io/vulnerability.manageiq" => {
```

Comment:
@cben maybe we need to use some "global" function that will return the name of this tool, "manageiq" or "cfme" ? but I didn't know where to one and using "cfme" here seemed wrong.